pvacseq/lib/main.py

```diff
@@ -190,6 +196,9 @@ def main(args_input = sys.argv[1:]):
     if args.fasta_size%2 != 0:
         sys.exit("The fasta size needs to be an even number")
 
+    if args.iedb_retries > 100:
+        sys.exit("The number of IEDB retries must be less than or equal to 100")
```

Comment:
Yeah, it probably warrants a separate PR if we make this change to all of them.  I was originally going to suggest using `choice` with a `range`, but it'd be silly to print out all the numbers between 0 and 100!